src/BlazorUI/Bit.BlazorUI.Tests/Buttons/BitToggleButtonTests.cs

```diff
@@ -4,7 +4,7 @@
 namespace Bit.BlazorUI.Tests.Buttons;
 
 [TestClass]
-public class BitToggleButtonTests : BunitTestContext
+public class BitToggleButtons : BunitTestContext
```

Comment:
This is the same as before, you should put the "Tests" at the end of the class name.

Reply:
That was on me.. done.